pkgs/development/python-modules/azure-mgmt-commerce/default.nix

```diff
@@ -10,13 +10,13 @@
 
 buildPythonPackage {
   pname = "azure-mgmt-commerce";
-  version = "1.0.1";
+  version = "1.8.2";
 
   src = fetchFromGitHub {
     owner = "Azure";
     repo = "azure-sdk-for-python";
-    rev = "ee5b47525d6c1eae3b1fd5f65b0421eab62a6e6f";
-    sha256 = "0xzdn7da5c3q5knh033vbsqk36vwbm75cx8vf10x0yj58krb4kn4";
+    rev = "azure-core_${version}";
+    sha256 = "1nx0mwkvwy34dpmkp3gbbjxvz12cnj2xr8lrxg1hq9clqkrl9yl5";
   };
```

Comment:
hmm, large mono repos....

Reply:
easiest solution I could think of would be to blacklist the repo. Or instead of making a "one-off" exception, just convert these to using `fetchPypi`

Reply:
there's only this and one other azure-mgmt package which are affected by it

thoughts? @FRidh

Reply:
I'm leaning towards just using fetchpypi, as I don't think tests are ran anyway